sample-app/config/initializers/action_mailer.rb

```diff
@@ -1,3 +1,3 @@
 options = {}
-ActionMailer::Base.add_delivery_method :ses, Aws::ActionMailer::SESMailer, **options
-ActionMailer::Base.add_delivery_method :ses_v2, Aws::ActionMailer::SESV2Mailer, **options
+ActionMailer::Base.add_delivery_method :ses, Aws::ActionMailer::SES::Mailer, **options
```

Comment:
isn't this an unrelated to mailbox change

Reply:
It's in the sample-app. A change I made for the other gem. I just forgot to change it here, and when testing ActionMailbox, I got an unrelated error.